Session/Conversations/ConversationVC.swift

```diff
@@ -1669,6 +1681,16 @@ final class ConversationVC: BaseVC, SessionUtilRespondingViewController, Convers
             self?.markFullyVisibleAndOlderCellsAsRead(interactionInfo: focusedInteractionInfo)
             self?.highlightCellIfNeeded(interactionId: focusedInteractionInfo.id, behaviour: behaviour)
         }
+
+        self.onScrollFinished()
+    }
+
+    func onScrollFinished() {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
Do we perhaps want to add the older code from `scrollViewDidEndDragging` and `scrollViewDidEndScrollingAnimation` to `onScrollFinished` as well? I'm not entirely sure they need to be fired every time scrolling is complete, though :)